packages/better-php-doc-parser/src/PhpDocNode/Symfony/SymfonyRouteTagValueNode.php

```diff
@@ -118,9 +118,8 @@ public function __toString(): string
             'path' => $this->printValueWithOptionalQuotes('path', $this->path, $this->localizedPaths),
         ];
 
-        if ($this->name) {
```

Comment:
Maybe `if ($this->name !== null) {`?

Reply:
I'm not sure if annotation parser from Doctrine makes any difference about these 2 cases.

Why do you need empty attribute?

Reply:
That's the point of this issue, isn't it? Name may be empty, but in this case an empty name is removed.

Reply:
I mean why do you need empty attribute in your code.

```php
$this->redirect('')
```

It might not be technically possible to differentiate, so if that has no effect, it should be removed in your code.

Reply:
No, a method `@Route` can have an empty name (`""`, not `null`) if the class `@Route` has a name.

Reply:
How can you use that?

Reply:
I'm not sure I understand you correctly. 

When you remove the name attribute from a method, the method is removed from routing. It should be empty, not removed. See my example: https://github.com/rectorphp/rector/pull/3240/files

That file contains valid code and should'nt be changed.

Reply:
See #3248